### PR TITLE
Avoid the usage of `shell=True` in Popen, which could lead to potential security risks.

### PR DESCRIPTION
**Pull request by misl6**
See full story here:
https://github.com/kivy/buildozer/issues/1433#issuecomment-1112906681
https://github.com/kivy/buildozer/issues/1433#issuecomment-1113074469
https://github.com/kivy/buildozer/issues/1433#issuecomment-1113958798

### CODE REVIEW

File: buildozer/targets/android.py [hunk 1]
```diff
@@ -569,7 +564,7 @@ def _android_update_sdk(self, *sdkmanager_commands):
             sdkmanager_path = self.sdkmanager_path
             sdk_root = f"--sdk_root={android_sdk_dir}"
             command = f"{yes_command} | {sdkmanager_path} {sdk_root} --licenses"
-            self.buildozer.cmd(command, cwd=self.android_sdk_dir)
+            self.buildozer.cmd(command, cwd=self.android_sdk_dir, shell=True)
```
Comment (tshirtman) [inline buildozer/targets/android.py:567]: i believe this one could be replaced using pipes to capture stdout and inject it in stdin of sdkmanager, but indeed our abstraction probably doesn't make that convenient.

**Review by tshirtman — APPROVED** [review]

File: buildozer/targets/android.py (hunk 1 above)
Comment (misl6) [reply buildozer/targets/android.py:564]: Yeah, that was a quick fix in order to see the CI running. I expect to remove it before the merge.

We have another `shell=True` on iOS, that needs some attention, but doesn't make use of anything that is injectable by the user via `buildozer.spec`, so is low-prio.

Comment (misl6) [reply buildozer/targets/android.py]: Changed this one, but kept the one on iOS.

**Comment by misl6** [comment]
🏁  I think is ready for a review.
✅ I did some additional tests locally and everything looks fine (at least on Android).

**Comment by misl6** [comment]
@tshirtman since you started a review on it, do you see anything that could lock us to merge it?

**Review by tshirtman — APPROVED** [review]
Looks good to me, didn't run it, (but tests & CI should catch any issue), and can't see anything wrong with the changes.

**Pull request merged**